src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    key={todo.id}
+    data-id={todo.id}
+    className={classNames('TodoInfo', {
+      'TodoInfo--completed': todo.completed,
+    })}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    <UserInfo user={todo.user} />
```

Comment:
user can be null
<img width="563" alt="image" src="https://github.com/user-attachments/assets/cea97d88-8b47-4a9d-b20f-e35c263baea6">

let's add the check
```suggestion
    {todo.user && <UserInfo user={todo.user} />}
```